Source/VisualStudio/vsnet2005.project.cst

```diff
@@ -42,6 +44,10 @@
 <%@ Property Name="WinLibNameSpace" Type="System.String" Category="Data" Description="Win Library Namespace." %>
 <%@ Property Name="IncludeWinLibrary" Type="System.Boolean" Category="Option" Description="" default="false" %>
 
+<%@ Property Name="IncludeSqlSprocs" Type="System.Boolean" Default="False" Category="Option" Description="" %>
+<%@ Property Name="IncludeSqlViews" Type="System.Boolean" Default="False" Category="Option" Description="" %>
+<%@ Property Name="IncludeSqlTables" Type="System.Boolean" Default="False" Category="Option" Description="" %>
+

```

Comment:
I assume the options here are for the SQL dump functionality.  You might include a description as to what specifically this option is for.  Maybe something like: "Dump Store Procedures to Files".  IncludeSqlProcs is a little bit vague.

Reply:
Ah, now I see why I did not add the descriptions there.  Because those are sub templates.  I always wondered why you guys put so many descriptions in the sub templates.  Was it common for people to try and run the templates individually without the master template?

Reply:
I added descriptions